src/command/list/human.rs

```diff
@@ -1,16 +1,25 @@
 //! Define the "human" format style for list commands.
 
-use textwrap::Wrapper;
+use textwrap::{HyphenSplitter, Wrapper};
 
-use volta_core::style::text_width;
+use volta_core::style::{text_width, tool_version, MAX_WIDTH};
 
-use super::{Node, Package, PackageManager, Source, Toolchain};
+use super::{Node, Package, PackageManager, Toolchain};
+
+use lazy_static::lazy_static;
 
 static INDENTATION: &'static str = "    ";
 static NO_RUNTIME: &'static str = "⚡️ No Node runtimes installed!
```

Comment:
Very minor nit, but I believe these types can be written without the `'static`, since static vars have that lifetime automatically (clippy gives a warning about this IIRC).

```suggestion
static INDENTATION: &str = "    ";
static NO_RUNTIME: &str = "⚡️ No Node runtimes installed!
```